app/Notifications/PostTagged.php

```diff
@@ -71,52 +85,36 @@ public function via($notifiable)
     }
 
     /**
-     * Get the mail representation of the notification.
+     * Get the Slack representation of the notification.
      *
-     * @param  mixed  $notifiable
-     * @return \Illuminate\Notifications\Messages\MailMessage
+     * @param mixed $notifiable
+     * @return \Illuminate\Notifications\Messages\SlackMessage
      */
     public function toSlack($notifiable)
     {
-        if (! config('services.slack.url')) {
-            return;
-        }
-
-        // Get the user & admin reviewer's names for the notification.
-        $data = app(GraphQL::class)->query(SLACK_NOTIFICATION_QUERY, [
-            'userId' => $this->post->northstar_id,
-            'adminId' => $this->adminId,
-        ]);
+        $data = $this->queryForUserAndReviewer();
 
         $userName = $data['user']['displayName'];
         $adminName = $data['admin']['displayName'];
 
         return (new SlackMessage)
             ->from('Rogue', ':tonguecat:')
-            ->content($adminName . ' just tagged this post as "' . $this->tag->tag_name . '":')
+            ->content($adminName.' just tagged this post as "'.$this->tag->tag_name . '":')
```

Comment:
This looks like a handy addition to https://github.com/DoSomething/communal-docs/tree/master/Contributing#php-code

Reply:
Oh ya. It should also be part of style checking.